handbook/engineering/hiring/software-engineer-frontend.md

```diff
@@ -2,15 +2,22 @@
 
 We are looking for frontend engineers who know how to build intuitive user experiences to make the power of [Universal Code Search](https://about.sourcegraph.com/product) accessible to everyone. You will have a lot of ownership to solve tough technical and UX problems in key areas of our web application and browser extensions (e.g. code search, code navigation, campaigns).
 
+The following engineering teams are currently hiring for this role:
```

Comment:
@tsenart FYI as discussed. Also, note the Hiring Manager interview step in the interview process and let me know if you are happy with this.

Reply:
LGTM, thanks!